packages/farms/constants/56.ts

```diff
@@ -50,6 +50,13 @@ const farms: SerializedFarmConfig[] = [
     quoteToken: bscTokens.wbnb,
   },
   //    * V3 by order of release (some may be out of PID order due to multiplier boost)
+  {
+    pid: 159,
+    lpSymbol: 'RDNT-BNB LP',
+    lpAddress: '0x9Ed9E9aA51670A3210fD6078024c21ec6c1d61d9',
```

Comment:
this

Reply:
to this 
0x346575fc7f07e6994d76199e41d13dc1575322e1